dev-scripts/vnet/cleanupworker-deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+docker build -t samo98/einvoice-cleanupworker -f Dockerfile-cleanupworker .
+docker push samo98/einvoice-cleanupworker
+"$JELASTIC_HOME"/environment/control/redeploycontainers --envName einvoice-apiserver-cluster --nodeGroup docker --tag latest
```

Comment:
envName has changed to `dev-einvoice-api-mfsr`.
Also how does this work? This is same command as notificationWorker redeploy